### ISSUE TITLE
[DOCS] OIDC (OpenID Connect) docs to contain common errors and solutions

### ISSUE THREAD

**Issue by InbarShimshon** [elastic/elasticsearch#61080]
https://www.elastic.co/guide/en/cloud/current/ec-secure-clusters-oidc.html#ec-oidc-user-settings

Example of an error that is pretty straightforward - though I think we need a more substantial list here:
```
Likely root cause: java.nio.file.NoSuchFileException: /usr/share/elasticsearch/config/oidc/jwkset.json at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) 
```

Resolution:
Populate this with a valid full path or https - in the yml file:
    xpack.security.authc.realms.oidc.okta.op.jwkset_path: oidc/jwkset.json

**_So 2 options here:_**
1) Set a full path and upload that jwkset.json file to your deployment, instructions here: https://www.elastic.co/guide/en/cloud/current/ec-secure-clusters-oidc.html#ec-oidc-user-settings ( step 4 )

2)Provide https, I see it would look something like this.
 jwk-set-uri: https://{yourOktaDomain}/oauth2/default/v1/keys
Save the changes on all nodes, restart the service and let us know how it goes!

I see we do provide "Common issues and solutions"  section for SAML e.g.: https://www.elastic.co/guide/en/elasticsearch/reference/master/trb-security-saml.html

Also - I opened this as a Feature Request with teams:doc , if this is not a correct path to open doc request please let me know

**Comment by jkakavas** [comment]
I agree we should have added this already. I think we can take it up in the security team to add a troubleshooting doc similar to saml
